source/geometries/Next100.cc

```diff
@@ -154,8 +154,10 @@ namespace nexus {
 
     G4ThreeVector gate_pos(0., 0., -gate_zpos_in_vessel_);
     if (lab_walls_){
-      G4ThreeVector castle_pos(0., hallA_walls_->GetLSCHallACastleY(),
```

Comment:
Why are you replacing the castle position, extracted from the geometry, with some hard-coded value?

Reply:
Not a real reason, just a remanent of the checks I was doing during the development...

Reply:
I would go back to the original version, then.